src/data-card-advanced.tsx

```diff
@@ -0,0 +1,463 @@
+                        onPress,
+                        label,
+                        Icon,
+                        iconSize = 20,
+                        iconColor = vars.colors.neutralHigh,
+                        iconBackground = styles.cardAction,
+                        iconBackgroundInverse = styles.cardActionInverse,
+                    },
+                    index
+                ) =>
+                    Icon ? (
+                        <IconButton
+                            size={48}
+                            key={index}
+                            onPress={onPress}
+                            aria-label={label}
+                            className={styles.cardActionIconButton}
+                            style={{display: 'flex'}}
+                        >
+                            <div className={isInverse ? iconBackgroundInverse : iconBackground}>
+                                <Icon color={iconColor} size={iconSize} />
+                            </div>
+                        </IconButton>
+                    ) : (
+                        <div key={index} className={styles.cardActionIconButton} />
+                    )
+            )}
+        </Inline>
+    );
+};
+
+const useTopActions = (actions?: Array<CardAction>, onClose?: () => void) => {
+    const {texts} = useTheme();
+    const finalActions = actions ? [...actions] : [];
+
+    if (onClose) {
+        finalActions.push({
+            label: texts.closeButtonLabel,
+            onPress: onClose,
+            Icon: IconCloseRegular,
+        });
+    }
+
+    return finalActions;
+};
+
+type CardContentProps = {
+    headline?: string | RendersNullableElement<typeof Tag>;
+    pretitle?: string;
+    pretitleAs?: string;
+    pretitleLinesMax?: number;
+    title?: string;
+    titleAs?: string;
+    titleLinesMax?: number;
+    subtitle?: string;
+    subtitleLinesMax?: number;
+    description?: string;
+    descriptionLinesMax?: number;
+};
+
+const CardContent: React.FC<CardContentProps> = ({
+    headline,
+    pretitle,
+    pretitleAs = 'h4',
+    pretitleLinesMax,
+    title,
+    titleAs = 'h3',
+    titleLinesMax,
+    subtitle,
+    subtitleLinesMax,
+    description,
+    descriptionLinesMax,
+}) => {
+    const {textPresets} = useTheme();
+    const renderHeadline = () => {
+        if (!headline) {
+            return null;
+        }
+        if (typeof headline === 'string') {
+            return <Tag type="promo">{headline}</Tag>;
+        }
+        return headline;
+    };
+    return (
+        <div>
+            <Stack space={4}>
+                {(headline || pretitle || title || subtitle) && (
+                    <header>
+                        <Stack space={4}>
+                            {renderHeadline()}
+                            <Stack space={4}>
+                                {pretitle && (
+                                    <Text2
+                                        aria-label="Texto"
```

Comment:
what?

Reply:
the aria label is necessary, since it was requested to already have a text in the aria label, in the accessibility part of figma this is documented